protos/encodings.proto

```diff
@@ -211,6 +211,14 @@ message BitpackedForNonNeg {
   Buffer buffer = 3;
 }
 
+message Bitpack2 {
+  // the number of bits of the uncompressed value. e.g. for a u32, this will be 32
+  uint64 uncompressed_bits_per_value = 2;
+
+  // The items in the list
+  Buffer buffer = 3;
```

Comment:
We probably don't need `buffer`.  That can be a concern for structural encodings (e.g. `PageLayout`).

Reply:
fixed. Thanks!